From acbd705b5e67183bb4e1800542816e1f997ab6ad Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Fri, 12 May 2023 16:50:31 +0200
Subject: [PATCH 0198/1141] arm64: defconfig: add stm32 timers support in
 addons fragment

Enable timer drivers that can be used on STM32MP25, as module.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I22b864a424a63fc15a4bcda886cdcd64d1eb76af
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/308362
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index eac5293f722c..1669995669aa 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -8,7 +8,9 @@ CONFIG_DWMAC_STM32=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_I2C_STM32F7=y
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
+CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_MFD_STM32_LPTIMER=m
+CONFIG_MFD_STM32_TIMERS=m
 CONFIG_NET_ACT_POLICE=m
 CONFIG_NET_ACT_SKBEDIT=m
 CONFIG_NET_ACT_VLAN=m
@@ -20,6 +22,7 @@ CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
+CONFIG_PWM_STM32=m
 CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
@@ -29,6 +32,7 @@ CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_SPI_STM32=m
 CONFIG_STM32_DMA3=y
 CONFIG_STM32_LPTIMER_CNT=m
+CONFIG_STM32_TIMER_CNT=m
 CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
 CONFIG_USB_DWC3_STM32=y
-- 
2.39.2

